ensime_shared/util.py

```diff
@@ -0,0 +1,35 @@
+import os
+from contextlib import contextmanager
+
+class Util:
```

Comment:
I haven't created Util, it was already defined https://github.com/ensime/ensime-vim/blob/master/ensime_launcher/__init__.py#L7